e2e/Tests/helpers/templates/FaultInjection.cs

```diff
@@ -78,15 +79,9 @@ internal static bool FaultShouldDisconnect(string faultType)
 
                 await deviceClient.SendTelemetryAsync(faultInjectionMessage, linkedCts.Token).ConfigureAwait(false);
             }
-            catch (IotHubClientException ex) when (ex.IsTransient || ex.ErrorCode == IotHubClientErrorCode.ConnectionForcefullyClosedOnFaultInjection)
```

Comment:
This was showing up in GWv2, but they said they'd roll it back, so we don't need it anymore.